Automattic/jetpack
# Social: Add pricing table to Jetpack Social

#### Changes proposed in this Pull Request:

* Add a pricing table to Jetpack Social for sites that have the share limit enabled.

#### Other information:


#### Jetpack product discussion


1202647675190352-as-1202849147697167/f

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Create a new site, connect to Jetpack and make sure the Jetpack Social admin page doesn't show a pricing table.
* Enable share limits (D85993-code)
* Reload the Jetpack Social admin page and check that there's a pricing table present.
* Choose "Start for free" and confirm that the table disappears.
* Reload the page and confirm the table is still gone.
* Run `wp option update jetpack-social_show_pricing_page true`.
* Reload the page again, but this time choose the paid plan.
* Go through checkout, and confirm:
  * The pricing table is gone again.
  * The share limits meter is gone, and the cards are shown instead.
